mdtp-example/src/main/java/io/github/protocol/mdtp/examples/MdtpServerExample.java

```diff
@@ -0,0 +1,12 @@
+package io.github.protocol.mdtp.examples;
+
+import io.github.protocol.mdtp.server.MdtpServer;
+import io.github.protocol.mdtp.server.MdtpServerConfig;
+
+public class MdtpServerExample {
+    public static void main(String[] args) throws Exception {
+        MdtpServerConfig mdtpServerConfig  = new MdtpServerConfig().host("127.0.0.1").port(8080);
```

Comment:
We'd better keep port sync with MdtpClient. BTW, does mdtp have their famous port?

Reply:
It seems they don't have a specific famous port.